integration/pvtdata/pvtdata_test.go

```diff
@@ -905,6 +761,180 @@ var _ bool = Describe("PrivateData", func() {
 			})
 		})
 	})
+
+	Describe("marble APIs invocation and private data delivery", func() {
+		var (
+			newLifecycleChaincode nwo.Chaincode
+			testChaincode         chaincode
+		)
+
+		BeforeEach(func() {
+			By("setting up the network")
+			network = initThreeOrgsSetup(true)
+
+			newLifecycleChaincode = nwo.Chaincode{
+				Name:              "marblesp",
+				Version:           "1.0",
+				Path:              components.Build("github.com/hyperledger/fabric/integration/chaincode/marbles_private/cmd"),
+				Lang:              "binary",
+				PackageFile:       filepath.Join(network.RootDir, "marbles-pvtdata.tar.gz"),
+				Label:             "marbles-private-20",
+				SignaturePolicy:   `OR ('Org1MSP.member','Org2MSP.member', 'Org3MSP.member')`,
+				CollectionsConfig: collectionConfig("collections_config1.json"),
+				Sequence:          "1",
+			}
+
+			By("setting DeliverService limit to 1 to verify grpc concurrency control")
```

Comment:
It is not obvious how this test verifies the grpc concurrency control. From talking to Wenjian I now understand it is an implicit verification via the rest of the test, not something that is tested explicitly in this test.
It could be added to any test. I think that should be better explained in the By here.
Better yet, add this implicit verification to a simpler test, not a the more complex private data test, as it gets lost in the weeds.

Reply:
Thank you Dave. Talked offline. Since IT has only 2 packages (lifecycle and pvtdata) calling DeliverService, I will keep the concurrency limit test in pvtdata, but add a comment explaining why we are doing it.